docs/debian-conf.d/auth/01_exim4-config_vexim

```diff
@@ -0,0 +1,24 @@
+
+### 01_exim4-config_vexim
+#########################
+
+plain_login:
+        driver = plaintext
+        public_name = PLAIN
+        server_condition = "${if crypteq{$3}{${lookup mysql{ \
+                          SELECT crypt FROM users \
+                          WHERE username = '${quote_mysql:$2}' \
+                          }}}{yes}{no}}"
+        server_set_id = $2
+        server_advertise_condition = ${if eq{$tls_cipher}{}{}{*}}
```

Comment:
I guess these login drivers should basically match those specified in docs/configure. That means that the advertise_condition should also match. See https://github.com/avleen/vexim2/blob/master/docs/configure#L932 and below.